store/searchtest/post_layer.go

```diff
@@ -1813,8 +1818,25 @@ func testShouldNotReturnLinksEmbeddedInMarkdown(t *testing.T, th *SearchTestHelp
 	defer th.deleteUserPosts(th.User.Id)
 
 	params := &model.SearchParams{Terms: "wikipedia"}
-	results, err := th.Store.Post().SearchPostsInTeamForUser([]*model.SearchParams{params}, th.User.Id, th.Team.Id, 0, 20)
+	results, err := th.Store.Post().SearchPostsForUser([]*model.SearchParams{params}, th.User.Id, th.Team.Id, 0, 20)
 	require.NoError(t, err)
 
 	require.Len(t, results.Posts, 0)
 }
+
+func testSearchAcrossTeams(t *testing.T, th *SearchTestHelper) {
```

Comment:
This file seems to be collapsed due to the large amount of changes. This (and the one at the top) are the only relevant changes.